src/Illuminate/Auth/SessionGuard.php

```diff
@@ -615,6 +617,21 @@ protected function fireAuthenticatedEvent($user)
         }
     }
 
+    /**
+     * Fire the logout other devices event if the dispatcher is set.
+     *
+     * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
+     * @return void
+     */
+    protected function fireLogoutOtherDevicesEvent($user)
+    {
+        if (isset($this->events)) {
+            $this->events->dispatch(new Events\LogoutOtherDevices(
```

Comment:
Should the namespace be imported at the top - then just `LogoutOtherDevices()` here?

Reply:
I'm just following the other existing `fire{EventName}Method` structure in the class.